dev-tools/mage/platforms.go

```diff
@@ -27,46 +27,7 @@ import (
 // BuildPlatforms is a list of GOOS/GOARCH pairs supported by Go.
 // The list originated from 'go tool dist list -json'.
 var BuildPlatforms = BuildPlatformList{
-	{"android/386", CGOSupported},
```

Comment:
Why do we need to remove these?

Reply:
When packaging a docker image with "make release", by default it will compile across all listed platforms, about 20+ of them, which is very time consuming. We only need amd64.

Also, filebeat's CI tool is very complicated and the documentation is almost non-existent, I read the code and only found this method.

Reply:
Then how about keeping this local and not commiting it?